### ISSUE TITLE
Port to macOS

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#1097]
OpenDungeons is developed as a cross-platform application and currently works on Linux and Windows, might build easily on *BSD variants too. It used to build on OSX but we have currently no contributor familiar with this platform, and would therefore welcome contributions.

**Comment by oyvindln** [comment]
I don't think it's ever built on OS X, the OS X specific code is just leftovers from the ogre example application.

**Comment by hwoarangmy** [comment]
hey @oyvindln 
Nice to see you are still around ^^

**Comment by oyvindln** [comment]
Yeah I've thought about doing some work on the project again now that I have a new computer that can compile the project relatively fast.

**Comment by hwoarangmy** [comment]
Well, no pressure from me. But if you want to get back, you are more than welcome ^^

**Comment by Bertram25** [comment]
@oyvindln Welcome back, mate :)
